extra/external-worker-aws-cf-template.yaml

```diff
@@ -300,7 +344,136 @@ Resources:
               Ref: VolumeSize
             VolumeType: "gp2"
             DeleteOnTermination: "true"
+      UserData:
```

Comment:
Can we use a file as parameter or something like this? Might make it easier to read and easier to use parameters for various variables

Reply:
I can open an issue for this. I only touched the UserData for telegraf installation.

Reply:
issue #29